### PR TITLE
fix: return exception details

### PR DESCRIPTION
**Pull request by Bento007**
- #2807

### Reviewers
**Functional:** @danieljhegeman 

**Readability:** @nayib-jose-gloria 

---

## Changes
- modify exceptions to return details.

### CODE REVIEW

File: backend/corpora/common/utils/color_conversion_utils.py [hunk 1]
```diff
@@ -178,16 +178,16 @@ def convert_color_to_hex_format(unknown):
             elif all(0 <= ele <= 255 and isinstance(ele, int) for ele in unknown):
                 tup = tuple(unknown)
             else:
-                raise ColorFormatException("Unknown color iterable format!")
+                raise ColorFormatException(detail="Unknown color iterable format!")
```
Comment (danieljhegeman) [inline backend/corpora/common/utils/color_conversion_utils.py:181]: well now this is just going above and beyond! 👏 👏

File: backend/corpora/common/utils/color_conversion_utils.py [hunk 2]
```diff
@@ -178,16 +178,16 @@ def convert_color_to_hex_format(unknown):
...
         elif isinstance(unknown, str) and unknown.lower() in CSS4_NAMED_COLORS:
             return CSS4_NAMED_COLORS[unknown.lower()]
         elif isinstance(unknown, str) and HEX_COLOR_FORMAT.match(unknown):
             return unknown.lower()
         else:
-            raise ColorFormatException("Unknown color format type!")
+            raise ColorFormatException(detail="Unknown color format type!")
     except Exception as e:
-        raise ColorFormatException(e)
+        raise ColorFormatException() from e
```
Comment (danieljhegeman) [inline backend/corpora/common/utils/color_conversion_utils.py:190]: was not familiar with `raise... from` until now. 👍

**Review by danieljhegeman — APPROVED** [review]
🚀

File: backend/corpora/common/utils/color_conversion_utils.py (hunk 1 above)
Comment (danieljhegeman) [reply backend/corpora/common/utils/color_conversion_utils.py:181]: (not Curator API-related)

**Review by nayib-jose-gloria — APPROVED** [review]

**Pull request merged**